cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -104,6 +114,24 @@ <h3 class="h4">
 
 {% macro _text(text) %}
 <p class="u-mb0">
-    {{ text }}
+    {{ text.blob.source | safe }}
 </p>
 {% endmacro %}
+
+{# ==========================================================================
+
+   _topics()
+
+   ==========================================================================
+
+   Description:
+
+   Creates related metadata related topics markup
+
+   value: An object holding the tag information (Unused)
+
+   ========================================================================== #}
+
+{% macro _topics(list) %}
+    {{ _list( page.related_metadata_tags(request.GET), true ) }}
```

Comment:
the urls for these topic links don't route to the related page url. It uses the current pages url